genai/image_generation/img_customization_canny_edge.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google import genai
+
+from google.genai.types import ControlReferenceConfig, ControlReferenceImage, EditImageConfig, EditImageResponse, Image
+
+
+def canny_edge_customization(output_file: str) -> EditImageResponse:
+    # [START googlegenaisdk_imagen_canny_edge_customization]
+
+    client = genai.Client()
+
+    image = Image(gcs_uri="gs://cloud-samples-data/generative-ai/image/car_canny.png")
+
```

Comment:
It would be nice to have doc string with a online statement here
```suggestion
   # Creating a reference image with ...
```